src/trivia/libraries/trivia/trivia.go

```diff
@@ -0,0 +1,91 @@
+package trivia
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+	"trivia/libraries/prize"
+	"trivia/libraries/problems"
+)
+
+const limitSeconds = 60
+const reward = 10
+const penalty = 5
+
+// Trivia game
+func Trivia() {
+	timeLimit := flag.Int("limit", limitSeconds, "the time limit for the quiz in seconds")
+	flag.Parse()
+
+	problems, err := problems.GetProblems()
+	if err != nil {
+		exit(err)
```

Comment:
we would more typically just return and let the main exit to make sure if there's some cleanup above it happens